### PR TITLE
Balance_Changes

### PR DESCRIPTION
**Pull request by lovesmuffins**
Changes to travel boots, ancient apparition, invoker, and echo sabre.

Releasing these in small batches.

Click on three dots right of changes_1 for more details.

### CODE REVIEW

**Comment by lovesmuffins** [comment]
didn't mean to include merge pull request, simply ignore it.

File: game/scripts/npc/abilities/templar_assassin_refraction.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "17.0"
+    "AbilityCooldown"                                     "17.0 16.0 15.0 14.0 13.0 12.0" // OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/templar_assassin_refraction.txt]: Revert this, its already 0 cd downtime.

File: game/scripts/npc/abilities/templar_assassin_refraction.txt [hunk 2]
```diff
@@ -42,7 +42,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "20 40 60 80 180 280"
+        "bonus_damage"                                    "20 40 60 100 200 300" // OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/templar_assassin_refraction.txt]: Revert lvl 4 dmg buff. TA isn't a bad hero.

File: game/scripts/npc/abilities/templar_assassin_refraction.txt (hunk 1 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/templar_assassin_refraction.txt]: I'd like you to show me an example of it, cause no TA can manage refractions as it's completely up to the competence of the enemy. I'll change it back for now.

File: game/scripts/npc/abilities/templar_assassin_refraction.txt [hunk 3]
```diff
@@ -42,8 +42,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "20 40 60 100 200 300" // OAA
-      }
+        "bonus_damage"                                    "20 40 60 80 200 300"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/templar_assassin_refraction.txt]: You removed } here?

Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/templar_assassin_refraction.txt]: Yeah you did. That's why Travis is complaining.

Comment (lovesmuffins) [reply game/scripts/npc/abilities/templar_assassin_refraction.txt]: my bad lol

File: game/scripts/npc/abilities/phantom_lancer_juxtapose.txt [hunk 4]
```diff
@@ -49,12 +49,12 @@
       "07"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "illusion_damage_in_pct"                          "550"
+        "illusion_damage_in_pct"                          "550 550 550 550 500" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/phantom_lancer_juxtapose.txt]: PL isn't in the game btw.

Comment (lovesmuffins) [reply game/scripts/npc/abilities/phantom_lancer_juxtapose.txt]: Ye I know lol I just felt like editing code more lol

Comment (lovesmuffins) [reply game/scripts/npc/abilities/phantom_lancer_juxtapose.txt]: w/o actually changing anything xd

File: game/scripts/npc/abilities/ancient_apparition_ice_blast.txt [hunk 5]
```diff
@@ -27,15 +27,15 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "40.0"
+    "AbilityCooldown"                                     "60.0" //OAA
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/ancient_apparition_ice_blast.txt]: the rest of the values are already pretty rough for AA, but this cooldown is a super overnerf. I think you *either* change the cooldown like this *or* you change the values in the way you did (and maybe add 5-10s, not 20s)

File: game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt [hunk 6]
```diff
@@ -41,12 +41,12 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "chance_pct"                                      "10 15 20 25 25 25"
+        "chance_pct"                                      "25"
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt]: I don't like seeing the chance buffed here, but I'm not opposed to a void buff. I just think it's the wrong part of him that i'd like to see become more powerful.

File: game/scripts/npc/abilities/templar_assassin_psi_blades.txt [hunk 7]
```diff
@@ -23,7 +23,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_attack_range"                              "60 120 180 240 300 360"
+        "bonus_attack_range"                              "60 120 180 240 320 400" // OAA
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/templar_assassin_psi_blades.txt]: TA doesn't need a buff, she's amazing. At the very least, she doesn't need a buff while the phase boots interactions are still in the game.

File: game/scripts/npc/items/farming/item_greater_travel_boots.txt [hunk 8]
```diff
@@ -48,8 +48,7 @@
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCooldown"                                     "45.0"
     "AbilitySharedCooldown"                               "travel"
-    "AbilityChannelTime"                                  "2.75 2.5 2.25 2"
-
+    "AbilityChannelTime"                                  "2"
```
Comment (chrisinajar) [inline game/scripts/npc/items/farming/item_greater_travel_boots.txt]: I like buffing travels, I think it's a great idea, but I don't like that it removes scaling. Boots are already hard to convince people to upgrade with the only exceptions being tranqs/arcanes for obvious reasons. Perhaps make something else about it scale nicely, or maybe just bump all the levels over by 1? `2.5 2.25 2 1.75`?

File: game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt (hunk 6 above)
Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt]: This buff is fine, makes his early game stronger. But if you actually want to make void good without changing time lock, increase his agi gain and lower his BAT.

File: game/scripts/npc/abilities/ancient_apparition_ice_blast.txt (hunk 5 above)
Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/ancient_apparition_ice_blast.txt]: Please dont nerf AA in any way. He is fine right now. His spells should be strong

File: game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt (hunk 6 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt]: I think agi gain might be hard-coded either way I wouldn't be the one capable on knowing how to do that at this moment.

Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt]: Jakiro had is agi gain buffed.

File: game/scripts/npc/abilities/ancient_apparition_ice_blast.txt (hunk 5 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/ancient_apparition_ice_blast.txt]: He is absolutely not fine as he is lmao. It's general consensus he's the best hero in the game right now.

**Pull request merged**